eth/core/state/interfaces.go

```diff
@@ -123,4 +123,13 @@ type (
 		// `ContainsAddress` returns whether the given `address` is in the access list.
 		ContainsAddress(address common.Address) bool
 	}
+
+	TransientJournal interface {
```

Comment:
```suggestion
	TransientStorageJournal interface {
```

This is more accurate and specific in my opinion.

Reply:
I agree & also renamed the previous transietStorage to be transietState since that felt more accurate